src/modules/url-changer.js

```diff
@@ -275,7 +275,13 @@
 	};
 	
 	var init = function () {
-		//Detect good strategy
+		//Init langs arrays
+		var html = $('html');
+		var languages = html.attr('data-all-langs');
+		if (!!languages) {
+			langs = languages.split(',');
```

Comment:
where is langs declared ? Should be `languages = languages.split(',');` ? I would add a else clause to set languages to `[]`